### ISSUE TITLE
RCS1016 .editorconfig

### ISSUE THREAD

**Issue by silvioboehme** [JosefPihrt/Roslynator#681]
**Product and Version Used**: 
VS2019 (16.6.1)
<PackageReference Include="Roslynator.Analyzers" Version="2.3.0">

I have found a strange behavior.
If i configure the analyzers in a ruleset file and set RCS1016 to "warning", VS2019 (16.6.1) shows this correcty as a warning for a simple one line method and provides the right fix.

If i configure the analyzers in .editorconfig and set RCS1016 to "warning", nothing happens.
Other analyzers are enabled in the right way, and also the severity seems to be right.

Except for RCS1016.

**Comment by JosefPihrt** [comment]
I will look at it.

It is important to mention that Roslynator does not interact with .editorconfig but it uses Roslyn API that allows to obtain effective severity for each analyzer. Which implies that theoretically it could be bug in Roslyn.

**Comment by caiolmd** [comment]
I also found a problem trying to disable a fade out analyzer through editorconfig. That is not working and the fading is still visible:

`dotnet_diagnostic.RCS1058.severity = none`
`dotnet_diagnostic.RCS1058FadeOut.severity = none`

**Comment by JosefPihrt** [comment]
@caiolmd "FadeOut" analyzers are marked with [NotConfigurable](https://github.com/dotnet/roslyn/blob/6ab11eedb13ef7c8d41f39313fd60c6622e48ba0/src/Compilers/Core/Portable/Diagnostic/WellKnownDiagnosticTags.cs#L41) tag which mean that they "cannot be suppressed or filtered or have its severity changed".

Are you trying to use RCS1058 without "fade out" effect? If that is the case then that is not possible.

**Comment by caiolmd** [comment]
> Are you trying to use RCS1058 without "fade out" effect? If that is the case then that is not possible.

@JosefPihrt No, I'm trying to disable RCS1058, which I did, but the "fade out" is still there. RCS1058 is disable but RCS1058FadeOut is not. I'm on VSCode with Omnisharp.

![image](https://user-images.githubusercontent.com/19883457/91107822-9e8a7b00-e64c-11ea-98d0-db15dd2dec8f.png)

**Comment by hntrewhitt** [comment]
Hey, I have experienced a similar fade out issue - when disabling the analyzer the fade out is still applied:

![image](https://user-images.githubusercontent.com/5013204/109840926-78b7e180-7c40-11eb-9fd8-f1905748f737.png)

![image](https://user-images.githubusercontent.com/5013204/109840983-840b0d00-7c40-11eb-8c8d-b2dd93cee17a.png)

But only for the switch statement, if behaves differently:

![image](https://user-images.githubusercontent.com/5013204/109841300-cb919900-7c40-11eb-8b5a-0527f52b5684.png)
